### PR TITLE
Create Veritaseum.md

### PR DESCRIPTION
**Pull request by Desser**

### CODE REVIEW

**Comment by akhavr** [comment]
Please add files with `people` to this PR

**Comment by Desser** [comment]
Ready

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Welcome to KeyCryptoVC Base
   		  
  - [Projects](projects/projects.md)  
- - People		  
+ - [People](projects/people.md)
```
Comment (akhavr) [inline README.md]: Why link to `projects/people.md` if you put changes into `people/people.md`?

**Comment by Desser** [comment]
Correction

**Pull request closed without merging**